### PR TITLE
Comment form doesn't allow click inside after comment submit

### LINKED ISSUE #1326: Comment form doesn't allow click inside after comment submit
**Describe the bug**
Comment form is closed and can't be opened again after sending a comment. It's waiting for a click outside probably

**To Reproduce**
1. Go to a goal
2. Click on comment form
3. Send a comment
4. Click on comment form

**Expected behavior**
Comment form should stay open and focused after sending comment

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes

- [x] Bug Fix

## Related issues

Resolve #1326 

## QA Instructions, Screenshots, Recordings
  

https://github.com/taskany-inc/issues/assets/95316053/1b96f812-9528-4715-a921-ba177824e84f

### CODE REVIEW

File: src/components/CommentCreateForm/CommentCreateForm.tsx [hunk 1]
```diff
@@ -33,9 +33,9 @@ const CommentCreateForm: React.FC<CommentCreateFormProps> = ({ goalId, states, o
     const [pushState, setPushState] = useState<State | undefined>();
 
     const createComment = useCallback(
-        (form: GoalCommentSchema) => {
+        async (form: GoalCommentSchema) => {
             // FIXME: maybe async/await would be better API
```
Comment (9teen90nine) [inline src/components/CommentCreateForm/CommentCreateForm.tsx:37]: probably you could remove this comment

**Review by 9teen90nine — APPROVED** [review]

**Review by awinogradov — CHANGES_REQUESTED** [review]
Now

image


Must be

image

**Comment by DenisVorop** [comment]
https://github.com/taskany-inc/issues/assets/95316053/fafe1f0f-938a-49fd-b6ee-59b0b950a55a

**Comment by DenisVorop** [comment]
https://github.com/taskany-inc/issues/assets/95316053/68e9db36-f982-413a-b4d8-b03f3a4742b5

**Comment by DenisVorop** [comment]
the latest video

https://github.com/taskany-inc/issues/assets/95316053/217cab0a-df8c-452c-8f04-65467e0f986c

**Review by awinogradov — APPROVED** [review]

**Pull request merged**